editor/project_manager.cpp

```diff
@@ -473,8 +473,20 @@ class ProjectDialog : public ConfirmationDialog {
 						cd->grab_focus();
 						return;
 					}
+					PackedStringArray project_features = ProjectSettings::get_required_features();
 					ProjectSettings::CustomMap initial_settings;
-					initial_settings["rendering/vulkan/rendering/back_end"] = rasterizer_button_group->get_pressed_button()->get_meta(SNAME("driver_name"));
+					// Be sure to change this code if/when renderers are changed.
+					int renderer_type = rasterizer_button_group->get_pressed_button()->get_meta(SNAME("driver_name"));
+					initial_settings["rendering/vulkan/rendering/back_end"] = renderer_type;
+					if (renderer_type == 0) {
+						project_features.push_back("Vulkan 1 Clustered");
+					} else if (renderer_type == 1) {
+						project_features.push_back("Vulkan 1 Mobile");
+					} else {
+						WARN_PRINT("Unknown renderer type. Please report this as a bug on GitHub.");
```

Comment:
This shouldn't be hardcoded (it's no longer accurate either).

Reply:
Do you happen to know where the enum is?

Reply:
Looks like it's hard-coded here too (`servers/rendering_server.cpp:2827`):

```
GLOBAL_DEF_RST_BASIC("rendering/vulkan/rendering/back_end", 0);
GLOBAL_DEF_RST_BASIC("rendering/vulkan/rendering/back_end.mobile", 1);
ProjectSettings::get_singleton()->set_custom_property_info("rendering/vulkan/rendering/back_end",
		PropertyInfo(Variant::INT,
				"rendering/vulkan/rendering/back_end",
				PROPERTY_HINT_ENUM, "Forward Clustered (Supports Desktop Only),Forward Mobile (Supports Desktop and Mobile)"));
```

Reply:
Hm right, I guess it's fine for now. We can rethink this once the OpenGL backend is ready and needs to be handled too.